### PR TITLE
Fix/10minute timeout action

### PR DESCRIPTION
**Pull request by ndelangen**
issue: https://github.com/chromaui/chromatic-cli/issues/273

### CODE REVIEW

File: action/main.ts [hunk 1]
```diff
@@ -148,19 +145,24 @@ async function run() {
...
+    setOutput('buildUrl', output.buildUrl);
+    setOutput('storybookUrl', output.storybookUrl);
+    setOutput('code', output.code.toString());
 
-    if (code !== 0) {
+    if (output.code !== 0) {
       setFailed('non-zero exit code');
     }
+
+    process.exit(output.code);
```
Comment (ndelangen) [inline action/main.ts:159]: here's the fix

Comment (ndelangen) [reply action/main.ts:159]: inspired from:
https://github.com/chromaui/chromatic-cli/blob/906f4426fc2641e78b556b1c1117d9b1e5301866/bin/main.js#L33-L36

**Comment by ndelangen** [comment]
Looks like it's running in 2m consistency..

File: action/main.ts [hunk 2]
```diff
@@ -148,19 +146,24 @@ async function run() {
       ignoreLastBuildOnBranch: maybe(ignoreLastBuildOnBranch),
     });
 
-    const [{ url, code }] = await Promise.all([chromatic]);
+    const output = await chromatic;
```
Comment (ghengeveld) [inline action/main.ts]: We don't need the `chromatic` variable here, just do `await runChromatic` directly.

File: action/main.ts [hunk 3]
```diff
@@ -148,19 +146,22 @@ async function run() {
...
     }
+
+    process.exit(output.code);
   } catch (e) {
     if (e.message) error(e.message);
     if (e.stack) error(e.stack);
     if (e.description) error(e.description);
     setFailed(e.message);
   }
+  process.exit(1);
```
Comment (ghengeveld) [inline action/main.ts:165]: I think this should be inside the `catch` block.

**Review by ghengeveld — APPROVED** [review]
LGTM

**Pull request merged**